### DISCUSSION TITLE
How to connect wireless clients to ap?

### DISCUSSION THREAD

**Discussion by deanfourie1** [netbox-community/netbox#15939] [Q&A]
I'm trying to connect wireless clients to access points,

I have managed to do this using the wireless links sections, however I can only connect a device to a APs wireless interface once, after that it is greyed out. Is there a way to connect multiple clients to say the one wlan 2g interface?

Or am I doing this wrong?

Thanks

**Comment by deanfourie1** [comment]
Again, this is such a weak function to be missing. Whats going on?

**Comment by deanfourie1** [comment]
I mean seriously, I really struggle to comprehend how this is still not implemented. I understand that netbox is an opensource platform and community, but to connect one-to-many wireless devices to an interface on an AP is so basic, and so logical, I just can't understand how this has not yet been implemented. I'm pulling my hair out here due to confusion.

If there is a logical reason, then please enlighten me by all means.

**Comment by ilumos** [comment]
This confused me too, but after reading the docs I learned wireless links are only for point-to-point links, and to model many WiFi clients/stations being associated with a WiFi SSID you've created:

1. Edit the WiFi client's interface
2. Set the **Type** to one of the 802.11 options
3. Scroll down to **Wireless**
4. Set the **Wireless role** to **Station**
5. In the **Wireless LANs** dropdown, add the SSID(s) the WiFi client is configured to connect to
6. Select **Save**

Then to see a list of stations and access points associated with the SSID:
1. Navigate to **Wireless** > **Wireless LANs** > **[your SSID name]**
2. Scroll down to **Attached Interfaces**

**Reply by diegogodoy06** [reply]
This is really interesting, the only problem is that I wouldn’t have this visualization in a topology? @deanfourie1

**Comment by tcutting63** [comment]
I similarly am struggling with modeling my wireless devices in my network:
As part of documenting my network, I want to include wireless devices.  I have added wifi interfaces to the devices, and they are connected to the appropriate Wireless LAN.
HOWEVER, these devices are not shown as "connected" - the options under "CONNECTIONS" do not support viewing these connections, and if I generate a network topology, these devices do not appear unless I filter  to "Show Unconnected: True".
It seems there should be a way to capture the wireless devices as actually connected to the network.
